packages/mobile/src/screens/universal-swap/index.tsx

```diff
@@ -212,49 +214,42 @@ export const UniversalSwapScreen: FunctionComponent = observer(() => {
     }
   });
 
-  // const isFocused = useIsFocused();
-
   const [isSelectFromTokenModal, setIsSelectFromTokenModal] = useState(false);
   const [isSelectToTokenModal, setIsSelectToTokenModal] = useState(false);
   const [isNetworkModal, setIsNetworkModal] = useState(false);
   const styles = styling(colors);
 
   const loadTokenAmounts = useLoadTokens(universalSwapStore);
   // handle fetch all tokens of all chains
-  const handleFetchAmounts = async (accounts) => {
+  const handleFetchAmounts = async () => {
     let loadTokenParams = {};
     try {
-      if (
-        accounts?.[ChainIdEnum.TRON] &&
-        accounts?.[ChainIdEnum.Ethereum] &&
-        accountOrai.bech32Address &&
-        accounts?.[ChainIdEnum.Oraichain] &&
-        accounts?.[ChainIdEnum.Injective]
-      ) {
-        const cwStargate = {
-          account: accountOrai,
-          chainId: ChainIdEnum.Oraichain,
-          rpc: oraichainNetwork.rpc
-        };
-        loadTokenParams = {
-          ...loadTokenParams,
-          oraiAddress: accounts[ChainIdEnum.Oraichain],
-          cwStargate
-        };
-        loadTokenParams = {
-          ...loadTokenParams,
-          metamaskAddress: accounts[ChainIdEnum.Ethereum]
-        };
-        loadTokenParams = {
-          ...loadTokenParams,
-          kwtAddress: getAddress(accounts[ChainIdEnum.Injective], 'oraie')
-        };
-        loadTokenParams = {
-          ...loadTokenParams,
-          tronAddress: accounts[ChainIdEnum.TRON]
-        };
+      const cwStargate = {
```

Comment:
![Screen Shot 2024-02-21 at 14 08 12](https://github.com/oraichain/owallet/assets/54303398/851231a3-8061-4e42-8518-b4764219b3b8)

I think it should be shortened